app/forms/applicants/open_banking_consent_form.rb

```diff
@@ -0,0 +1,9 @@
+module Applicants
+  class OpenBankingConsentForm
+    include BaseForm
+
+    form_for LegalAidApplication
+
+    attr_accessor :open_banking_consent, :consent_choice_timestamp
```

Comment:
`consent_choice_timestamp` isn't used by this form as it is set via a callback in the model. So just:
```ruby
attr_accessor :open_banking_consent
```